### DISCUSSION TITLE
Failed to use raisimPy in Windows10

### DISCUSSION THREAD

**Discussion by emreutku** [raisimTech/raisimLib#296] [Q&A]
Hello all,
I followed the installation steps in the raisim official page for windows installation. I had created a virtual environment using Anaconda and gave the python directory of it to the installation command like:

cmake .. -DCMAKE_INSTALL_PREFIX=$LOCAL_INSTALL -DRAISIM_EXAMPLE=ON -DRAISIM_PY=ON -DPYTHON_EXECUTABLE=C:\Users\MYUSERNAME\Anaconda3\envs\raisim_ws\python.exe

After that I added "C:\Users\MYUSERNAME\Desktop\Raisim_ws\raisimLib\build\$LOCAL_INSTALL\bin" to Path system variable to make any python to find my raisim, basically.

Then, when I activate my conda environment and try to import raisimpy it cannot find it.

Do you have any idea how to handle this?

Thanks in advance
Cheers!

**Comment by jhwangbo** [comment]
is this a correct path: "C:\Users\MYUSERNAME\Desktop\Raisim_ws\raisimLib\build$LOCAL_INSTALL\bin"

**Reply by emreutku** [reply]
No it is a typo. The one I added to path is:
"C:\Users\MYUSERNAME\Desktop\Raisim_ws\raisimLib\build\ $LOCAL_INSTALL\bin"
I should add the bin directory, shouldnt I?

**Reply by jhwangbo** [reply]
you have to check if "raisimpy.cp3..." file exists in the directory that you specified. Make sure that the version matches in the file name

**Reply by emreutku** [reply]
![image](https://user-images.githubusercontent.com/51458858/163563174-8d4ce5e4-15e8-4db5-9bba-563cadbbdd4b.png)
Do you mean these files? They are present in the path location.

**Reply by jhwangbo** [reply]
I don't think $LOCAL_INSTALL works. Write the actual path not the environment variable
